technical-docs/rc-releng.md

```diff
@@ -20,12 +20,12 @@ The repository also includes a set of commands for publishing both the standalon
 3. Select the appropriate version bump when prompted via the console. If the operation is aborted at this point, only `lerna publish` is needed to be run again to continue.
 4. The core packages should be published to the public NPM registry and internal references should be updated. Note: By default, `lerna publish` will create a git tag that points to this branch, so in a later step we must regenerate this tag to point to the release commit on master.
 5. Run `npm run publish:demo-server` to publish the demo application to the template repository, it will be published under a branch called `release-{semver in lerna.json}`
-6. If alignment with a platform release is necessary, clone the https://github.com/feedhenry-raincatcher/raincatcher-server repository and copy the new branch to one with the required name:
+6. Clone the https://github.com/feedhenry-raincatcher/raincatcher-server repository and copy the new branch to one with the required name:
 
 ```
 git clone git@github.com:feedhenry-raincatcher/raincatcher-server.git
 cd raincatcher-server/
-git push origin release-{semver}:FH-{RHMAP version}
+git push origin +release-{semver}:master
```

Comment:
Is the `+` here the same as `--force`?

Reply:
Yes. Never use force. Never ever. Use it only if you want upstream to look like your local repo :D